README.md

```diff
@@ -481,29 +481,11 @@ Install to the workbench's virtualenv by running the following command form the
 pip install -r requirements.txt
 ```
 
-In the main XBlock repository, create the following configuration file
-in `workbench/settings_mentoring.py` in the XBlock repository:
-
-```python
-from settings import *
-
-INSTALLED_APPS += ('mentoring',)
-DATABASES['default']['NAME'] = 'workbench.sqlite'
-```
-
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
-
-```bash
-$ ./manage.py syncdb --settings=workbench.settings_mentoring
```

Comment:
@antoviaque 

Yes, after looking into this more, it's clear I will need to leave the Django models in place for now, at least in order to migrate the student data. (They can be kept separate and only need to be installed and used for migration purposes).

For accessing data across students I actually was just about to ask you about that. There are two options:

1. Keep using the Answers django model.
2. Convert to using standard XBlock storage as I have done so far - however, in order to do the data export that approach would require this one piece of code that uses non-XBlock APIs:

```python
    def _get_students_data(self, answer_block):
        """
        Efficiently query for the answers entered by ALL students.
        (Note: The XBlock API only allows querying for the current
        student, so we have to use other APIs)

        Yields tuples of (student_id, student_answer)
        """
        usage_id = answer_block.scope_ids.usage_id
        # Check if we're in edX:
        try:
            from courseware.models import StudentModule
            usage_id = usage_id.for_branch(None).version_agnostic()
            entries = StudentModule.objects.filter(module_state_key=unicode(usage_id)).values('student_id', 'state')
            for entry in entries:
                state = json.loads(entry['state'])
                if 'student_input_raw' in state:
                    yield (entry['student_id'], state['student_input_raw'])
        except ImportError:
            pass
        # Check if we're in the XBlock SDK:
        try:
            from workbench.models import XBlockState
            for entry in XBlockState.objects.filter(scope="usage", scope_id=usage_id).exclude(user_id=None).values('user_id', 'state'):
                state = json.loads(entry['state'])
                if 'student_input_raw' in state:
                    yield (entry['user_id'], state['student_input_raw'])
        except ImportError:
            pass
        # Something else - return only the data
        # for the current user.
        yield (answer_block.scope_ids.user_id, answer_block.student_input_raw)
```

So the question is, which is worse - this code above, or requiring a separate Django model just so we can do data export?